proposals/lip-0049.md

```diff
@@ -113,32 +113,28 @@ The following constants are used throughout the document, multiple of those cons
 | Name                                           | Type   | Value  |
```

Comment:
If LIP 45 is already the source of truth for constants, I do not see the value to replicate them here.